lib/go-tc/parameters.go

```diff
@@ -277,12 +277,15 @@ func ParamExists(id int64, tx *sql.Tx) (bool, error) {
 
 // ParamsExist returns whether parameters exist for all the given ids, and any error.
 // TODO move to helper package.
-func ParamsExist(ids []int64, tx *sql.Tx) (bool, error) {
-	count := 0
-	if err := tx.QueryRow(`SELECT count(*) from parameter where id = ANY($1)`, pq.Array(ids)).Scan(&count); err != nil {
-		return false, errors.New("querying parameters existence from id: " + err.Error())
+func ParamsExist(ids []int64, tx *sql.Tx) (bool, []int64, error) {
+	var nonExistingIDs []int64
+	if err := tx.QueryRow(`SELECT ARRAY_AGG(id) FROM UNNEST($1::INT[]) AS id WHERE id NOT IN (SELECT id FROM parameter)`, pq.Array(ids)).Scan(pq.Array(&nonExistingIDs)); err != nil {
+		return false, nil, errors.New("querying parameters existence from id: " + err.Error())
```

Comment:
constructing errors from errors should use wrapping e.g. with `fmt.Errorf``'s `%w` formatting parameter.

Reply:
Also, this error shouldn't be shown to the user, but the way this works currently it would be. You don't need to fix that here, but you can if you want.

Reply:
I understand that we shouldn't since it is a DB level. but also there is a `TODO` to rework validation in `traffic_ops/traffic_ops_golang/api/api.go`. So, I would do it as a separate PR.
```// TODO: Rework validation to be able to return system-level errors
type ParseValidator interface {
	Validate(tx *sql.Tx) error
}```